src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -1591,7 +1591,7 @@ class C
         bool x = await [|M()|].ConfigureAwait(false);
     }
 }";
-            await TestAsync(text, "global::System.Threading.Tasks.Task<System.Boolean>");
+            await TestAsync(text, "global::System.Threading.Tasks.Task<System.Boolean>", testPosition: false);
```

Comment:
because at the start position of that expression (which is what "testPosition" is testing) there is no reason to infer Task<bool> if you have a member access expression.